src/Umbraco.Core/Services/MemberService.cs

```diff
@@ -941,19 +936,25 @@ public void Delete(IMember member)
         {
             using (var uow = UowProvider.GetUnitOfWork())
             {
-                if (uow.Events.DispatchCancelable(Deleting, this, new DeleteEventArgs<IMember>(member)))
-                {
-                    uow.Commit();
-                    return;
-                }
-
-                var repository = RepositoryFactory.CreateMemberRepository(uow);
-                repository.Delete(member);
+                Delete(uow, member);

```

Comment:
I'm just passing the uow to the `Delete` method so that it can do what it has to do, but it will not commit, so I have to commit afterwards. Making sense?